uiowa/uiowa
# Narrow article feature

Resolves https://github.com/uiowa/uiowa/issues/6128. 


# How to test 
```
ddev blt frontend && ddev blt ds --site=ilr.law.uiowa.edu && ddev drush @lawilr.local config-split:activate articles_narrow -y && ddev drush @lawilr.local uli
```

`ddev blt frontend`
`ddev blt ds --site=ilr.law.uiowa.edu`
`ddev drush @lawilr.local config-split:activate articles_narrow`
`ddev drush @lawilr.local uli`

- Code review
  - Test show/hide functionality of image block and broken links block.  I had to rewrite that code to get it to work with an additional layout.
- User story "As a Law Journal website owner, I would like to see our articles in a narrow container to increase readability for users."
- Verify that law articles are rendering with 67/33 sidebar.  All meta info should be in sidebar (screenshots below in details).
- Sync the sandbox site and verify that articles remain the same. 
- Test articles list, featured content list, /news list and verify that I didn't mess up the styling there. 
- Test mobile/desktop all breakpoints.

<details>

<img width="1691" alt="Screenshot 2023-02-13 at 9 16 46 AM" src="https://user-images.githubusercontent.com/1036433/218497095-6ba50c61-55b9-44c9-8d78-66f20d74984b.png">
<img width="1587" alt="Screenshot 2023-02-13 at 9 16 04 AM" src="https://user-images.githubusercontent.com/1036433/218497098-9bce1f89-e770-4f76-8da0-4e395f63fa91.png">

</details>
